askama_parser/benches/from_str/librustdoc/LICENSE

```diff
@@ -0,0 +1,7 @@
+The files in this folder were copied verbatim from [librustdoc/html/templates] in version
+eeb59f16a5f40e14dc29b95155b7f2569329e3ec. Dual licensed under MIT OR Apache-2.0.
+
+Please find the authors in [their Git history].
+
+[librustdoc/html/templates]: <https://github.com/rust-lang/rust/tree/master/src/librustdoc/html/templates>
+[their Git history]: <https://github.com/rust-lang/rust/commits/master/src/librustdoc/html/templates>
```

Comment:
@GuillaumeGomez, do you think this file is good enough, or do you think we need to add a shortlog?

Reply:
What do you mean?

Reply:
Whether a reference to the commit in the folder is enough, or if I should actually mention all contribution. I decided to err on side of caution: <https://github.com/djc/askama/blob/5de051cfe32fe8468d1d5d96bf458a07df5170e6/askama_parser/benches/from_str/librustdoc/LICENSE.md>